chatbot-ui/src/components/ChatInterface.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './ChatInterface.css'; // Import CSS for styling
+import { v4 as uuidv4 } from 'uuid';
+import maskot from '../assets/maskot.png';
+
+const ChatInterface = () => {
+  const [messages, setMessages] = useState([]);
+  const [input, setInput] = useState('');
+  const [sessionId, setSessionId] = useState(''); // Add state for sessionId
+
+  // Generate a new sessionId when the component mounts
+  useEffect(() => {
+    setSessionId(uuidv4());
```

Comment:
What do we need the sessionID for?

Reply:
session info will be required, when we will deploy salesGPT to internet to store conversation history for each user